monitoring/mock_uss/run_locally_riddp.sh

```diff
@@ -10,7 +10,13 @@ AUD=${MOCK_USS_TOKEN_AUDIENCE:-localhost,host.docker.internal}
 
 PORT=8073
 
-docker run --name mock_uss_riddp \
+if [ "$CI" == "true" ]; then
+  docker_args="--add-host host.docker.internal:host-gateway" # Required to reach other containers in Ubuntu (used for Github Actions)
+else
+  docker_args=""
+fi
+
+docker run "${docker_args}" --name mock_uss_riddp \
```

Comment:
Quotes around ${docker_args} is breaking the script (MacOS) when the value is empty.
Error: `docker: invalid reference format.`

Reply:
Added `shellcheck disable=SC2086` (not sure how best to avoid this finding otherwise)